README.md

```diff
@@ -1,2 +1,23 @@
-# .github
-Organization README etc
+![OpsHelm Banner - Secure Your Cloud by Default](opshelm-github-hero-banner.png)
+
+
+## Make Classes of Security Problems Impossible
+
+The OpsHelm team is working hard behind the scenes to help you automate your cloud infrastructure security remediation. It goes beyond identifying cloud misconfigurations — it fixes them in seconds.
+
+<style>
```

Comment:
Does Github support CSS in their Markdown? This doesn't seem to be being parsed and is just being dumped in the output, AFAICT.

Reply:
You're meant to be able to in the markdown standard, but I see what you mean, shall update 👍🏻